README.md

```diff
@@ -38,56 +36,59 @@ Obs: Please note that Apple Clang Version number does not follow upstream LLVM/C
 
 #### Ubuntu 22.04
 
-```
+```shell
 sudo apt-get install build-essential automake libtool patchelf cmake pkg-config wget git libreadline-dev libboost-coroutine-dev libboost-context-dev libboost-filesystem-dev libssl-dev openssl libc-ares-dev zlib1g-dev ca-certificates liblua5.3-dev
```

Comment:
```suggestion
sudo apt-get install build-essential automake libtool patchelf cmake pkg-config wget git libreadline-dev libboost-coroutine-dev libboost-context-dev libboost-filesystem-dev liboost-log-dev libcrypto++-dev libssl-dev openssl libc-ares-dev zlib1g-dev ca-certificates liblua5.3-dev
```

I had to add those packages to solve some compilation errors, but I still get the following error:

```
 make
make[1]: Entering directory '/home/endersonmaia/Development/cartesi/server-manager/src'
g++ -fno-strict-aliasing -fno-strict-overflow -fno-delete-null-pointer-checks  -Wl,-rpath,'$ORIGIN/' -o server-manager versioning.pb.o cartesi-machine.pb.o cartesi-machine-checkin.pb.o cartesi-machine.grpc.pb.o cartesi-machine-checkin.grpc.pb.o server-manager.pb.o server-manager.grpc.pb.o health.pb.o health.grpc.pb.o complete-merkle-tree.o pristine-merkle-tree.o protobuf-util.o server-manager.o -lcryptopp -L/home/endersonmaia/Development/cartesi/server-manager/build/Linux_x86_64/lib -lgrpc++ -lgrpc -lgpr -lprotobuf -lpthread -labsl_synchronization -lboost_coroutine -lboost_context -lboost_log -lboost_log_setup -lboost_thread -ldl
/usr/bin/ld: /home/endersonmaia/Development/cartesi/server-manager/build/Linux_x86_64/lib/libgrpc.so: undefined reference to `SSL_get_peer_certificate'
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:316: server-manager] Error 1
make[1]: Leaving directory '/home/endersonmaia/Development/cartesi/server-manager/src'
make: *** [Makefile:118: source-default] Error 2
```

Not sure if this PR is the scope for a fix like that, or maybe a separate commit in this PR.